assets/js/googlesitekit/datastore/site/connection.js

```diff
@@ -129,6 +131,16 @@ export const reducer = ( state, { type, payload } ) => {
 export const resolvers = {
 	*getConnection() {
 		try {
+			const registry = yield Data.commonActions.getRegistry();
+
+			const existingConnection = registry.select( STORE_NAME ).getConnection();
+
+			// If there are already accounts loaded in state, we don't want to make this request
```

Comment:
Copy pasta comment 😄 
```suggestion
			// If there is already connection data loaded in state, don't make this request
```